plugins/ReactNativeStyle/StyleEdit.js

```diff
@@ -39,7 +39,7 @@ class StyleEdit extends React.Component {
 
   onChange(name: string, val: string | number) {
     var num = Number(val);
-    this.props.onChange(name, num == val ? num : val);
+    this.props.onChange(name, num === val ? num : val);
```

Comment:
This looks like it will change behavior in an unintended way; since `val` can be either a string or number, the `==` check was probably intentional (since eg `4 == "4"`, and we are dealing with user input here). If flow isn't happy about the typing here, then maybe we need to coerce one of the results of the ternary into either a string or number.